data-shaping/output/output.py

```diff
@@ -0,0 +1,669 @@
+                "hei_extension": hei_extension,
+                "hei_extension_date": hei_extension_date,
+                "hei_extension_date_comments": hei_extension_date_comments,
+                "program_certification": program_certification,
+                "program_certification_date": program_certification_date,
+                "program_leaving_date": program_leaving_date,
+                "program_leaving_reason": program_leaving_reason,
+                "program_leaving_comments": program_leaving_comments,
+                "hsst_pathway_id": hsst_pathway,
+                "hsst_portfolio_completed": hsst_portfolio_completed,
+                "hsst_portfolio_completion_date": hsst_portfolio_completion_date,
+                "hsst_expected_exit_date": hsst_expected_exit,
+                "hsst_arp_completed": hsst_arp_completed,
+                "hsst_arp_completion_date": hsst_arp_completion_date,
+                "hsst_d_clin_part_a_completed": hsst_d_clin_part_a_completed,
+                "hsst_d_clin_part_a_completion_date": hsst_d_clin_part_a_completion_date,
+                "hsst_d_clin_part_b_completed": hsst_d_clin_part_b_completed,
+                "hsst_d_clin_part_b_completion_date": hsst_d_clin_part_b_completion_date,
+                "hsst_d_clin_part_c1_completed": hsst_d_clin_part_c1_completed,
+                "hsst_d_clin_part_c1_completion_date": hsst_d_clin_part_c1_completion_date,
+                "hsst_d_clin_part_c2_completed": hsst_d_clin_part_c2_completed,
+                "hsst_d_clin_part_c2_completion_date": hsst_d_clin_part_c2_completion_date,
+                "hsst_fcrpath_completed": hsst_fcrpath_completed,
+                "hsst_fcrpath_completion_date": hsst_fcrpath_completion_date,
+                "hsst_iaps_completed": hsst_iaps_completed,
+                "hsst_iaps_completion_date": hsst_iaps_completion_date,
+                "hsst_phd_completed": hsst_phd_completed,
+                "hsst_phd_completion_date": hsst_phd_completion_date,
+                "hsst_ceng_completed": hsst_ceng_completed,
+                "hsst_ceng_completion_date": hsst_ceng_completion_date,
+                "hsst_portfolio_signed": hsst_portfolio_signed,
+                "hsst_portfolio_signed_date": hsst_portfolio_signed_date,
+                "hsst_start_month": [hsst_start_month],
+                "hsst_start_year": [hsst_start_year],
+                "hsst_expected_completion_month": [hsst_expected_completion_month],
+                "hsst_expected_completion_year": [hsst_expected_completion_year],
+                "reasonable_adjustments": reasonable_adjustments,
+                "reasonable_adjustments_comments": reasonable_adjustments_comments,
+                "stp_start_month": [stp_start_month],
+                "stp_start_year": [stp_start_year],
+                "stp_expected_completion_month": [stp_expected_completion_month],
+                "stp_expected_completion_year": [stp_expected_completion_year],
```

Comment:
`start_month`
`start_year`
`expected_completion_month`
`expected_completion_year`

as per our chat recently
🙏

Reply:
so drop the prefixes

Reply:
wasn't the last chat where we agreed to keep them? As it was easier to manage? I can change anyways, just wanted to check.

Reply:
due to the way UI is implemented - it's quite a headache to have separate values for each programme....
so unprefixed is better...